src/Json/Exception/JsonValidationException.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Json\Exception;
+
+use JsonSchema\Exception\ValidationException;
+
+/**
+ * @internal
+ */
+final class JsonValidationException extends ValidationException
+{
+    /**
+     * @param string $path
```

Comment:
I think we can safely remove this doc, what do you think?

Reply:
@BackEndTea @sidz 

Perhaps we should enable the `no_superfluous_phpdoc_tags` fixer?

Reply:
agree :) This came from the my current project where doc block is required everytime :)